{"name": "用户登录--预期失败--用例二", "status": "passed", "attachments": [{"name": "log", "source": "5205cd6b-d998-4cd3-9ab1-da790cf6f977-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "''"}, {"name": "password", "value": "''"}, {"name": "expect_alert", "value": "['请输入用户名', '请输入密码']"}], "start": 1678676351373, "stop": 1678676352668, "uuid": "1c145afa-2881-4490-94af-f95b96baf97b", "historyId": "a26968bce4e5c5d61bb6a44c3deb2d36", "testCaseId": "db3e6fbd6ca68f97daf4eb9e50ba75ee", "fullName": "scripts.test01_mp_login.TestMpLogin#test_mp_login_error_2", "labels": [{"name": "severity", "value": "blocker"}, {"name": "feature", "value": "用户登录"}, {"name": "epic", "value": "针对业务场景的测试"}, {"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test01_mp_login"}, {"name": "subSuite", "value": "TestMpLogin"}, {"name": "host", "value": "wht"}, {"name": "thread", "value": "6988-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test01_mp_login"}]}